tensorboard/program_test.py

```diff
@@ -62,6 +62,18 @@ def testMakeServerBlankHost(self):
         self._StubApplication(),
         self.make_flags(host='', port=0, path_prefix=''))
     self.assertStartsWith(server.get_url(), 'http://')
+    
+    def testPathPrefixSlash(self):
+    #Test that checks the path prefix ends with one trailing slash
+    server = program.WerkzeugServer(
+        self._StubApplication(),
+        self.make_flags(host='127.0.0.1', port=0, path_prefix='/test'))
```

Comment:
Rather than setting the host, let's just use `''` like the test above so this works on IPv4-only and IPv6-only stacks.

Same below.

Reply:
Ohk!